setup.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+import 'dotenv/config';
+import { User } from './src/moduls/User.js';
```

Comment:
There is a typo in the import path: 'moduls' should be 'modules'. Please correct the path to './src/modules/User.js'.